inference/core/env.py

```diff
@@ -106,7 +106,7 @@
 
 # Flag to enable SAM core model, default is True
 CORE_MODEL_SAM_ENABLED = str2bool(os.getenv("CORE_MODEL_SAM_ENABLED", True))
-CORE_MODEL_SAM2_ENABLED = str2bool(os.getenv("CORE_MODEL_SAM2_ENABLED", False))
+CORE_MODEL_SAM2_ENABLED = str2bool(os.getenv("CORE_MODEL_SAM2_ENABLED", True))
```

Comment:
do we want to support SAM2 at hosted platform within all other core models?

Reply:
That's the goal

Reply:
then we must:
* add remote execution into workflows block
* add functional tests for the suite of platform tests

Reply:
We changed plans on this. For now (maybe forever) we want to ship SAM2 with support for GPU mainly (OK to allow enabling on CPU via env var).  But for hosted deployments we will not support SAM2 on lambda.  Want to use it to promote new dedicated inference server feature Shuyang has built.  Working on UI for that in workflows next week (will label certain blocks as needing GPU and suggest / let you spin up dedicated dev GPU for building in workflows UI)

Reply:
We can still decide later to support it on hosted if we do more benchmarking and implement remote execution for block.